apps/web/src/components/templates/in-app-editor/AvatarFeedFields.tsx

```diff
@@ -0,0 +1,136 @@
+import { useInputState } from '@mantine/hooks';
+import { ActionIcon, Divider } from '@mantine/core';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useEffect, useState } from 'react';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { showNotification } from '@mantine/notifications';
+import { IFeedEntity } from '@novu/shared';
+
+import { Checkbox, colors, Input } from '../../../design-system';
+import { useEnvController } from '../../../store/useEnvController';
+import { createFeed, getFeeds } from '../../../api/feeds';
+import { QueryKeys } from '../../../api/query.keys';
+import { PlusGradient } from '../../../design-system/icons';
+import { FeedItems } from './FeedItems';
+import { EnableAvatarSwitch } from './EnableAvatarSwitch';
+
+export const AvatarFeedFields = ({ index, control }) => {
```

Comment:
I struggle to understand what's the relation with Avatars for this component besides the use of `EnableAvatarSwitch` inside. What is this component goal?

Reply:
It is a grouping for avatar fields and feed fields